src/website/src/app/documentation/demos/a11y/accessibility.demo.html

```diff
@@ -0,0 +1,67 @@
+<clr-doc-wrapper [ng]="ng" [ui]="ui" [title]="title" [newLayout]="newLayout">
+  <article>
+
+    <div id="guidelines">
+      <h3 id="accessibility">Accessibility</h3>
+      <p>
+        Although Clarity tries to cover as much as possible all best practices for being a аccessibility compliant. Some cases
+        are too domain-specific so to get the best experience there some tools below that are designed to help you.
+      </p>
+
+      <h3 id="aria-live-region">Aria live region</h3>
+      <p>
+        The possibility of having a dynamic component that is changing the content, notifications or loading states could create
+        a content that could not be obvious for some users. To improve this we could use assistive technologies. ARIA live regions
+        fill this gap and provide a way to programmatically expose dynamic content changes in a way that can be announced by
+        assistive technology.
+      </p>
+
+      <p>
+        There is a limitation on how different assistive technology readers are handling the <code class="clr-code">aria-live</code> region. So we provide a
+        service to handle this type of case.
+      </p>
+
+      <p>
+        From <strong>V3</strong> there is a <code class="clr-code">AriaLiveService</code> as part of the accessibility module. The service will let you create a live
+        region elements inside the DOM so screen readers could track and announce there content to the users.
+      </p>
+
+      <clr-code-snippet [clrCode]="example1" [clrLanguage]="'typescript'"></clr-code-snippet>
+
+      <p>
+        Aria live region could be set as one of three politness level <code class="clr-code">polite</code> by default this is the standart type of messages
+        they will be announced one after the other in the order they are register, <code class="clr-code">assertive</code> will cancel previous
+        messages or any other announcment that is active at the moment and broadcast itself, <code class="clr-code">off</code> no message will be broadcasted.
+      </p>
+      <clr-code-snippet [clrCode]="example2" [clrLanguage]="'typescript'"></clr-code-snippet>
+
+      <h5 id="aria-live-cleanup">Aria live cleanup</h5>
+
+      <p>
```

Comment:
Remove the whole `Aria live cleanup` section, people shouldn't have to do this at the app level.